odysseus/odysseus_tree/overlays/rootfs_overlay_nrc_common/usr/bin/build_nrc_params.py

```diff
@@ -212,13 +213,9 @@ def run_common():
     insmod_arg = setModuleParam(items)
 
     file_txt = ""
-    mod_path = ""
-    # if not passed in, use live buildroot default
-    if len(sys.argv) <= 3:
-        mod_path = "/etc/modprobe.d/nrc.conf"
-    else:
-        mod_path = sys.argv[3]
 
+    mod_path = args.mp
```

Comment:
can you add a default for mod_path

Reply:
nvm

Reply:
default option is set in 
parser.add_argument('--mp', help="path/to/modprobe.d/directive.conf (optional, otherwise default: /etc/modprobe.d/nrc.conf)", default="/etc/modprobe.d/nrc.conf")